brick/brick.go

```diff
@@ -0,0 +1,88 @@
+package brick
+
+import (
+	"crypto/md5"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/gluster/glusterd2/utils"
+	"github.com/gluster/glusterd2/volume"
+)
+
+const (
+	// TODO: Remove hardcoding
+	//	glusterfsd = "/usr/local/sbin/glusterfsd"
+	glusterfsd = "/usr/bin/sleep"
+)
+
+type Brick struct {
+	// Externally consumable using methods of Daemon interface
+	binarypath     string
+	args           string
+	socketfilepath string
+	pidfilepath    string
+
+	// For internal use
+	brickinfo volume.Brickinfo
```

Comment:
I intended to have this separation as `volume.Brickinfo` is part of volume package and I'd prefer each daemon to be in it's own package - hence the `brick` package. We could move contents of `BrickInfo` here and update wherever it's used but again, `Brickinfo` has only three fields and I don't see a reason for `volume` package to import `brick` package in it's code. I'd prefer them to be more modular with less dependency.

IMO, a general rule of thumb good to follow is any daemon package should only import "generic" packages like utils, context, config and errors. I'd prefer to remove the import and use of `volume` package here in future.